packages/x-charts/src/internals/plugins/featurePlugins/useChartCartesianAxis/useChartCartesianAxisRendering.selectors.ts

```diff
@@ -279,3 +358,72 @@ export const selectorChartRawAxis = createSelector(
     return axis;
   },
 );
+
+export const selectorChartDefaultXAxisId = createSelector(
+  [selectorChartRawXAxis],
+  (xAxes) => xAxes![0].id,
+);
+
+export const selectorChartDefaultYAxisId = createSelector(
+  [selectorChartRawYAxis],
+  (yAxes) => yAxes![0].id,
+);
+
+const EMPTY_MAP = new Map<SeriesId, Flatbush>();
+export const selectorChartSeriesEmptyFlatbushMap = () => EMPTY_MAP;
+
+export const selectorChartSeriesFlatbushMap = createSelector(
+  [
+    selectorChartSeriesProcessed,
+    selectorChartNormalizedXScales,
+    selectorChartNormalizedYScales,
```

Comment:
This has a bug. The normalized scales aren't responding to the changes in the filtered domains, so this isn't being re-created when the filtered domain changes, causing the wrong data points to be shown.